frontend/src/entities/jobs/algorithms.tsx

```diff
@@ -24,10 +24,12 @@ import deepARMetadata from '../../shared/algorithmMetadata/forecastingDeepAR.jso
 import object2Vec from '../../shared/algorithmMetadata/object2Vec.json';
 import imageClassification from '../../shared/algorithmMetadata/imageClassification.json';
 import factorizationMachines from '../../shared/algorithmMetadata/factorizationMachines.json';
+import semanticSegmentationMetadata from '../../shared/algorithmMetadata/semanticSegmentation.json';
 import React, { ReactNode } from 'react';
-import { floatIntervalValidator, integerIntervalValidator, IntervalStrategy } from '../../shared/validation/helpers/numbers';
```

Comment:
Are these helpers used at all now? I thought the outcome of our discussion was that we wanted validators we could use across the application and all of our validators would exist in this helper and then be referenced from here as well as various create forms. If that's not the case let's remove that helper file completely to avoid the issue that originally prompted this change which was that we had multiple validators that appeared to offer duplicate functionality and it wasn't clear when to use what.

Reply:
Bringing this up as a team discussion. I think the best path forward is to move the new on to this validator file for re-use. Clarifying direction with the team, but yes I will remove these functions too. They shouldn't be used elsewhere.